README.md

```diff
@@ -199,6 +559,12 @@ Currently, the crystallisation journey and tax account in staging can only be ac
 | --- | --- |
 | AA888888A | XAIT00000888888 |
 
+## Ninos with stub data for Income Tax Submission Frontend
+
+| Nino | Income Tax Submission Frontend data | Source |
+| --- | --- | --- |
```

Comment:
Can add couple more NINOs with different income sources when entering the service for both in year and EOY.